resources/todomvc/vanilla-examples/javascript-es6/dist/view.js

```diff
@@ -85,22 +89,18 @@ class View {
     _removeItem(id) {
         let elem = qs(`[data-id="${id}"]`);
 
-        if (elem) {
-            this.$todoList.removeChild(elem);
-        }
+        if (elem) this.$todoList.removeChild(elem);
     }
 
     _clearCompletedButton(completedCount, visible) {
-        this.$clearCompleted.innerHTML = this.template.clearCompletedButton(completedCount);
+        this.$clearCompleted.textContent = this.template.clearCompletedButton(completedCount);
```

Comment:
Similar than in ES5, I replaced this with using `textContent` for the same reason.